### PR TITLE
Rename SwapTimeoutTransaction -> SwapAbortTransaction

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: packages/iov-bns/src/decode.ts [hunk 1]
```diff
@@ -224,13 +224,13 @@ function parseSwapClaimTx(
   };
 }
 
-function parseSwapTimeoutTx(
+function parseSwapAbortTransaction(
```
Comment (willclarktech) [inline packages/iov-bns/src/decode.ts]: Do we have any rules about when to use `tx` and when to use `transaction`? This change is not consistent with how it was before or the `parseSwapClaimTx` function.

Comment (webmaster128) [reply packages/iov-bns/src/decode.ts]: My idea is to ban the term "tx" in all non-Tendermint code. It seems to be very popular in the Tendermint community but not necessarily in all blockchain world. This is why in BCP everything is *transaction but there is still a lot of legacy.

The whole naming build* and parse* in encode.ts/decode.ts is inconsitent. It should probably be "encode<transaction type>" and "decode<transaction type>". I started this renaming as part of this PR but it became too much so I stopped the internal renamings to focus on external facing changes only.

Comment (willclarktech) [reply packages/iov-bns/src/decode.ts]: OK, I'm happy to incrementally switch to `transaction` only.

**Comment by webmaster128** [comment]
> I find a few more occurrences

thanks, renamed

**Comment by ethanfrey** [comment]
If you are complaining about my internal function names, it means, you have tackled all my code.

Please feel free to rename it all. I just care that the functionality works with good test coverage. You two are much better at naming than I

**Comment by webmaster128** [comment]
No problem, we'll do Ethan :) It is much easier to cleanup existing functionality as clarity is improving than coming up with all that from scratch!

**Pull request merged**